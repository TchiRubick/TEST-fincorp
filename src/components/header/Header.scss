.comp-header {
  display: flex;
  
  & > * {
    margin-right: 1rem;
  }

  span {
    align-self: center;
    font-weight: bold;
  }
}