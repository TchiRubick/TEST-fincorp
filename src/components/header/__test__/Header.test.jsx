import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Header from '../Header';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<Header />, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
