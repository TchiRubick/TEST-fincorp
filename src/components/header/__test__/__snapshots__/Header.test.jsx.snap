// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-header"
>
  <img
    alt="logo"
    src="logo.png"
  />
  <span>
    For You & Family
  </span>
  <span>
    For Business
  </span>
</div>
`;
